### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by iamlasse**
Updates to profile page

### CODE REVIEW

File: client/templates/initiatives/mine/mine.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
       {{/each}}
     </div>
     <div class="initiative-mine row center-align">
-        <a class="button btn" href="{{pathFor 'initiatives.create'}}">New Initiative</a>
+        <a class="button waves-effect waves-teal btn-flat" href="{{pathFor 'initiatives.create'}}">New Initiative</a>
```
Comment (trustieee) [inline client/templates/initiatives/mine/mine.html]: Would you mind trying the following classes here to match the create initiative button on the create screen?

```
class="btn waves-effect waves-light blue darken-1"
```

or something similar perhaps?

Comment (iamlasse) [inline client/templates/initiatives/mine/mine.html]: Sure thing, i think the buttons are hideous lol eventually i'll do a complete design... but for now its ok

Comment (iamlasse) [inline client/templates/initiatives/mine/mine.html]: Fixed a few mobile size issues too

Comment (iamlasse) [inline client/templates/initiatives/mine/mine.html]: That worked, now we have counts in a central place!

**Pull request merged**